web/themes/custom/pineapple/sass/components/paragraphs/_image-paragraph.scss

```diff
@@ -1,14 +1,20 @@
-.paragraph--type--image-paragraph {
-  .image-paragraph-image {
-    img {
-      object-fit: cover;
-      width: 100%;
-    }
+.field--type-image {
```

Comment:
I think this class should be `.image-paragraph-container` because again the current class is added for every media entity that is an image